src/nvim/auevents.lua

```diff
@@ -19,6 +19,8 @@ return {
     'BufWriteCmd',            -- write buffer using command
     'BufWritePost',           -- after writing a buffer
     'BufWritePre',            -- before writing a buffer
+    'ChannelOpened',          -- channel was opened
+    'ChannelInfo',            -- info was received about channel
```

Comment:
Perhaps we should stick with "chan" for all user facing channel APIs. Instead of "channel" in some places but not others. 

Same for API function names.

Reply:
👍